deploy-operator/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Build the manager binary
```

Comment:
Let's name the binary `deploy-operator`. If we call it `manager`, that's confusing with `workflow-manager`.